pages-domain/src/main/java/nl/vpro/domain/page/RelationDefinitionService.java

```diff
@@ -11,12 +9,11 @@
 public interface RelationDefinitionService {
 
 
-    // TODO
+
+
+    @Deprecated
     static RelationDefinitionService getInstance() {
-        try {
-            return (RelationDefinitionService) Class.forName("nl.vpro.domain.page.RelationDefinitionServiceImpl").getMethod("getInstance").invoke(null); } catch (IllegalAccessException | InvocationTargetException | NoSuchMethodException | ClassNotFoundException e) {
-            throw new IllegalArgumentException(e);
-        }
+        return RelationDefinitionServiceProvider.getInstance();
```

Comment:
An interface should not have an getInstance method at all. One might however use a provider to obtain an instance.